### PR TITLE
1.7.5 update

### PR DESCRIPTION
**Pull request by Ma-rcx**
Leave this PR open for now until I update (most of) the stuff that changed.

### CODE REVIEW

**Comment by Ma-rcx** [comment]
Finished everything that 1.7.5 introduced, will now go over some mistakes that I've spotted and push the commit.

**Comment by Ma-rcx** [comment]
The last two commits finish this PR. @wofsauge please take a look and request changes if necessary.

Multiple items used the Tear Delay stat, which was [replaced with Fire Rate in Repentance](https://bindingofisaacrebirth.fandom.com/wiki/Tears#Direct_modifiers_to_Tear_Delay/Tears_Fired_per_Second) and this was changed in the descriptions to better reflect stat increases/decreases.

**Comment by kittenchilly** [comment]
Also something to note: Telescope lens gives +24% planetarium chance, up from the previously though +9%. Tested using the new GetPlanetariumChance function.

**Comment by Ma-rcx** [comment]
> 
> 
> Also something to note: Telescope lens gives +24% planetarium chance, up from the previously thought +9%. Tested using the new GetPlanetariumChance function.

Thanks! The original description didn't include a percentage, so I've updated it now using your number.

File: descriptions/rep/en_us.lua [hunk 1]
```diff
@@ -328,7 +328,7 @@ local repCollectibles={
 	[689] = {"689", "Glitched Crown", "Item pedestals quickly cycle between 5 random items"}, --  Glitched Crown
 	[690] = {"690", "Belly Jelly", "Causes enemies to bounce off of Isaac#Bouncing them against obstacles deals damage#Chance to deflect enemy projectiles"}, --  Belly Jelly
 	[691] = {"691", "Sacred Orb", "Prevents low-quality items from spawning#Greatly increasing quality of items that will spawn"}, --  Sacred Orb
-	[692] = {"692", "Sanguine Bond", "Spawns a set of spikes in the {{DevilRoom}}Devil Room#Taking damage from the spikes can grant a reward:#35%: 6 pennies#15%: +0.5 Damage up#5%: 2 Black Hearts#2%: Random item#1%: Leviathan transformation"}, --  Sanguine Bond
+	[692] = {"692", "Sanguine Bond", "Spawns a set of spikes in the {{DevilRoom}}Devil Room#Taking damage from the spikes can grant a reward:#35%: 6 pennies#15%: +0.5 Damage up#5%: 2 Black Hearts#2%: Random item#1%: Leviathan transformation"}, --  Sanguine Bond //TBD: Someone on the wiki said the percentages were changed and that's mentioned in the changelog. Would like if someone could check those.
```
Comment (Ma-rcx) [inline descriptions/rep/en_us.lua:331]: Someone mind checking this and getting the exact numbers?

**Comment by kittenchilly** [comment]
I was actually wrong about Telescope Lens again.
It raises the default chance from 1% to 15% (meaning it'll stay 15% if you've already entered a planetarium room instead of 1%), and adds a +9% chance on top of it

**Pull request merged**